src/components/Answer.tsx

```diff
@@ -45,23 +67,30 @@ export const Answer = (props: AnswerProps) => {
 
   let body, feedback, selectedCount;
 
-  const isChecked = isAnswerChecked(answer, chosenAnswer);
+  const isChecked = isAnswerChecked(answer, answerId);
   const isCorrect = isAnswerCorrect(answer, correctAnswerId);
   const isIncorrect = isAnswerIncorrect(answer, incorrectAnswerId);
-
-  const correctIncorrectIcon = (
-    <div className="correct-incorrect">
-      {isCorrect && props.correctIncorrectIcon}
-    </div>
-  );
+  // When rendering a previous response, we can determine if it was this answer.
+  // If there is no incorrectAnswerId, that means only a correct answer is present, check isCorrect.
+  // If an incorrectAnswerId is present (there is only ever one, if multiple attempts are enabled,
+  // it is the latest one) checking isIncorrect works because incorrectAnswerId is only set for
+  // a missed attempt, meaning if an attempt is missed and then successfully re-attempted,
+  // incorrectAnswerId will be empty.
+  const isPreviousResponse = answerId === undefined && (!incorrectAnswerId && isCorrect || isIncorrect);
```

Comment:
I'm guessing the answerId === undefined makes the previous answer disappear when you select a new one?

Reply:
Right, on load it should appear selected if there is no current selection, on selection it should follow the isChecked logic.